### PR TITLE
PMM-5655 remove leftovers of Grafana plugins

### PR DESCRIPTION
**Pull request by adivinho**
- [ ] https://github.com/Percona-Lab/pmm-submodules/pull/771

### CODE REVIEW

File: fix-panels.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            if os.path.exists(panel_path):
+                print '   * %r -> ' % (panel,),
+                with open(panel_path, 'r') as f:
+                    panel_params = json.loads(f.read())
+                    print '%r' % (panel_params['id'],)
+                    if (not(os.path.exists(os.path.join(GRAFANA_PLUGINS_DR, panel_params['id'])))):
+                        print '    * Renaming panel'
+                        shutil.copytree(os.path.join(GRAFANA_PLUGINS_DR, panel), os.path.join(GRAFANA_PLUGINS_DR, panel_params['id']))
+                    if (panel != panel_params['id']):
+                        print '    * Removing leftovers'
```
Comment (atymchuk) [inline fix-panels.py:28]: I'd suggest to call it `Cleaning up...` or something similar, since it's not clear why it a panel is a leftover.

**Comment by atymchuk** [comment]
@adivinho where does the script `fix-panels.py` get called?

**Comment by adivinho** [comment]
> @adivinho where does the script `fix-panels.py` get called?

It's called during preparing a docker

**Comment by AlekSi** [comment]
@adivinho IIRC, it is safe to merge it now. That should also fix pmm-update Travis CI tests.

**Pull request merged**